libmicroros.mk

```
@@ -64,6 +64,7 @@ $(EXTENSIONS_DIR)/micro_ros_src/src:
 	git clone -b master https://github.com/micro-ROS/rosidl_typesupport src/rosidl_typesupport; \
 	git clone -b main https://github.com/micro-ROS/rosidl_typesupport_microxrcedds src/rosidl_typesupport_microxrcedds; \
 	git clone -b rolling https://github.com/ros2/rosidl src/rosidl; \
+	git clone -b rolling https://github.com/Acuadros95/rosidl_dynamic_typesupport src/rosidl_dynamic_typesupport; \
```

```suggestion
	git clone -b rolling https://github.com/ros2/rosidl_dynamic_typesupport src/rosidl_dynamic_typesupport; \
```

Remember to modify this before merge